internal/prompter/prompter.go

```diff
@@ -49,11 +50,22 @@ type surveyPrompter struct {
 	stderr    io.Writer
 }
 
+// LatinMatchingFilter returns whether the value matches the input filter.
+// The strings are compared normalized in case and diacritics.
+func LatinMatchingFilter(filter, value string, index int) bool {
+	filter = strings.ToLower(text.RemoveDiacritics(filter))
```

Comment:
Let's keep the diacritics in the original filter string, and just ToLower it. If someone typed in `ø` wanting to search for that character specifically, we'd be doing them a disservice by matching that against a literal `o`.